atom-publisher-lib/src/main/scala/com/gu/atom/data/DataStore.scala

```diff
@@ -24,7 +24,6 @@ trait DataStore extends DataStoreResult {
   /* this will only allow the update if the version in atom is later
    * than the version stored in the database, otherwise it will report
    * it as a version conflict error */
-  def updateAtom(newAtom: Atom): DataStoreResult[Unit]
 
```

Comment:
if they have the same api but different implementations I think it makes sense to keep the method description in the parent interface. But I that's just my opinion, I am happy either way!